### PR TITLE
Clarify backend options, add missing options

### PR DESCRIPTION
**Pull request by ilumos**
Today I switched my dual-stack firewall over from the legacy `ddclient` backend to the `native` backend, and I hope this update to the docs will help others do the same with ease. Thank you

### CODE REVIEW

File: source/manual/dynamic_dns.rst [hunk 1]
```diff
@@ -20,54 +17,55 @@ use the [+] button to install it.
...
 ---------------------------
 
-In the primary tab you can register one or more dynamic dns providers which will be used to update dns registrations
-using an api call over http(s) to the selected service.
-
-.. Note::
-
-      The local IP address used for this firewall will be obtained by querying one of the selected providers. Since ddclient
-      currently doesn't support dual stack (IPv4+IPv6) opertion, make sure to either select an IPv4 or IPV6 address
-      provider in the settings tab.
```
Comment (AdSchellevis) [inline source/manual/dynamic_dns.rst:53]: we might want to leave some note between the relation of the "check ip method" and the protocol being used.

**Review by AdSchellevis — CHANGES_REQUESTED** [review]
@ilumos some small remarks, rest looks fine, thanks

**Comment by ilumos** [comment]
Thanks @AdSchellevis, I've implmenented your improvement suggestions.

Regarding the note on protocols, did you see lines 64 to 69, which clarifies how to set up for dual stack?

**Comment by AdSchellevis** [comment]
@ilumos I just did, thanks.

**Comment by ilumos** [comment]
Awesome, thank you!

**Pull request merged**